system/lvol.py

```diff
@@ -212,7 +221,11 @@ def main():
                 changed = True
             else:
                 lvcreate_cmd = module.get_bin_path("lvcreate", required=True)
-                rc, _, err = module.run_command("%s %s -n %s -%s %s%s %s" % (lvcreate_cmd, yesopt, lv, size_opt, size, size_unit, vg))
+                if opts is None:
```

Comment:
Yeah, I wasn't really in love with that as I was fixing the merge conflict. I'll fix that up real quick.